rpc/block_svc_routines.c

```diff
@@ -3783,6 +3782,7 @@ blockCreateCliFormatResponse(struct glfs *glfs, blockCreateCli *blk,
     blockFormatErrorResponse(CREATE_SRV, blk->json_resp, errCode,
                              GB_DEFAULT_ERRMSG, reply);
   }
+  reply->exit = errCode;
```

Comment:
For the BUG raised, this line fixes the problem. Create exit code is not collected before, so fixing the code to collect the right exit code.

Reply:
@pkalever If you look at the pull request/commit message it doesn't explain the problem what you encountered and how it is solving it. People generally do that when a patch is sent, otherwise the context will be forgotten after a few months. I just want you to capture that in the pull request.

Reply:
@pkalever Thanks for the commit message. I am seeing similar issue in modify, resize, info. I think even in Delete we can move it to the end like you did here. Did you get a chance to check those as well? Am I missing some other detail?

Reply:
Done, thanks!